pkgs/by-name/uv/uv-sort/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "uv-sort";
+  version = "0.5.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "uv_sort";
+    inherit version;
+    hash = "sha256-qCShDuKBFS4omcsntZ1wzRtAKTbp8CfTw0IIAgxBvcE=";
+  };
+
+  dependencies = with python3Packages; [
+    packaging
+    tomlkit
+    typer
+  ];
+
+  build-system = with python3Packages; [
+    hatchling
+    uv-dynamic-versioning
+  ];
+
+  nativeCheckInputs = with python3Packages; [
+    pytestCheckHook
+  ];
```

Comment:
nit and not blocking, but for just one input here you probably don't need to use `with` and it'll increase eval times to remove it